### PR TITLE
remove redundant values

### PR DESCRIPTION
**Pull request by pq**
See: https://github.com/flutter/devtools/commit/83b319ecb83a2b900363d83391384f584a317577#commitcomment-38174345.

/cc @jacob314 @kenzieschmoll @devoncarew

### CODE REVIEW

**Comment by devoncarew** [comment]
@RedBrogdon - perhaps a using lint to promote via devrel channels? We'd hit this issue in a code review, and didn't know about the existing lint ourself.

**Comment by devoncarew** [comment]
@pq, if you run `./bin/repo_tool analyze`, you'll be able to hit all the packages in the repo.

**Comment by devoncarew** [comment]
You may need to run `./bin/repo_tool package-get` first.

**Comment by pq** [comment]
Ah, thanks.  Running now! 👍

**Comment by pq** [comment]
(Sorry for all the noise here.  For some reason ` ./bin/repo_tool analyze` is running clean for me here despite errors picked up on the bots...)

File: packages/devtools_app/test/utils_test.dart [hunk 1]
```diff
@@ -14,12 +14,12 @@ void main() {
 
       expect(printMb(10 * MB, 0), '10');
       expect(printMb(10 * MB), '10.0');
-      expect(printMb(10 * MB, 1), '10.0');
```
Comment (jacob314) [inline packages/devtools_app/test/utils_test.dart:17]: ignore the lint for this case or remove as this line is now identical to the one above

File: packages/devtools_app/test/utils_test.dart [hunk 2]
```diff
@@ -14,12 +14,12 @@ void main() {
...
+      expect(printMb(10 * MB), '10.0');
       expect(printMb(10 * MB, 2), '10.00');
 
       expect(printMb(1000 * MB, 0), '1000');
       expect(printMb(1000 * MB), '1000.0');
-      expect(printMb(1000 * MB, 1), '1000.0');
```
Comment (jacob314) [inline packages/devtools_app/test/utils_test.dart:22]: ignore the lint for this case or remove as this line is now identical to the one above

**Comment by RedBrogdon** [comment]
@devoncarew Yes, that's a great idea. Looks like this one will be landing in 2.8.0? Is there any convenient way to see which new lints are going to land in an upcoming version?

**Pull request merged**